### ISSUE TITLE
Magnifying glass & wheelchair images on Search screen

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#600]
The magnifying glass and the wheelchair images are being picked up by screen readers. They should be marked as decorative images. 

![GitHub13](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/0e480abb-bac8-4046-9fc4-b25ac43020af)
